client/js/components/document/ElementsAdminList.vue

```diff
@@ -149,12 +149,12 @@ export default {
   },
 
   methods: {
-    ...mapActions('elements', {
+    ...mapActions('Elements', {
```

Comment:
When navigating to the page ("Planunterlagen und Planzeichnung"), I get the error 
`Error in mounted hook: "TypeError: Cannot read properties of undefined (reading 'then')"` and the elements are not loading

Reply:
b945b5b3d5bd6392bcad8cd2dab5481685dfd150